src/utils/updateValueWithTemplate.js

```diff
@@ -1,15 +1,20 @@
 import { isEmpty } from 'lodash';
 
 export const updateValueWithTemplate = (option, template) => {
-  const findPattern = /(?<=\*\*)(\w+)(?=\*\*)/g;
-  const replacePattern = /\*\*(\w+)\*\*/;
+  const findPattern = /\*\*(\w+)\*\*/g;
 
   let updatedString = template;
   const match = template.match(findPattern);
 
   if (!isEmpty(match)) {
     match.forEach(key => {
-      updatedString = updatedString.replace(replacePattern, option[key]);
+      const currentKey = key.replace(/\*\*/g, '');
+      const replacePattern = new RegExp(`\\**${currentKey}\\**`, 'g');
```

Comment:
Are you sure you need escape characters here?

Reply:
according to developer.mozilla.org, yes - 
![image](https://user-images.githubusercontent.com/40805351/83789811-e8904000-a69f-11ea-98be-294209f814b5.png)

Reply:
In this case, maybe it's needed to escape second asterisk as well?
```suggestion
      const replacePattern = new RegExp(`\\*\\*${currentKey}\\*\\*`, 'g');
```

Reply:
I've tested and it works the same in both cases..

Reply:
Did you try to match smth like this - `****code*****`?

Reply:
should we really cover such case?

Reply:
you're right, not the same. escaped second asterisk as well.